.github/workflows/test_pypy.yml

```diff
@@ -0,0 +1,44 @@
+name: PyPy
+
+on: push
```

Comment:
I think you need this to be:

```yaml
on:
    pull_request:
        branches: [master]
```

Or some variation on that.

Reply:
Right, that might make sense, my bad. It ran on the PRs on my repo, but it probably just reused the runs from my second branch. I'll try force push a change in a sec.